bfunwind/src/dwarf4.cpp

```diff
@@ -30,14 +30,22 @@
 template<typename T> T static get(char **p)
 { auto v = *(T *)(*p); (*p) += sizeof(T); return v;}
 
-#define if_opcode(a,b) \
+#define if_cfa(a,b) \
     if (opcode == a) \
     { \
         log("  %s: ", #a); \
         b \
         return; \
     }
 
+#define if_opcode(a,b) \
+    if (opcode == a) \
+    { \
+        log("    - %s: ", #a); \
+        b \
+        continue; \
```

Comment:
Why is this continue but if_cfa is return? Never a huge fan of hiding a branch or jump in a macro...